extensions/wikia/VideoHandlers/apiwrappers/ApiWrapper.class.php

```diff
@@ -600,6 +600,12 @@ public function __construct( $status, $content, $apiUrl ) {
 		}
 
 		$this->message = $message;
+
+		Wikia\Logger\WikiaLogger::instance()->error(
+			__CLASS__,
+			[ 'exception' => $this ]
```

Comment:
Let's log the `$apiUrl` and `$status->errors` as separate fields in the context. The `exception` field will be formatted by `Monolog` and contain exception message, code and backtrace only.